precompiles/parachain-staking/src/tests.rs

```diff
@@ -183,6 +187,126 @@ fn points_non_zero() {
 		});
 }
 
+#[test]
+fn delegation_amount_zero() {
+	ExtBuilder::default()
+		.with_balances(vec![(Alice.into(), 1_000)])
+		.build()
+		.execute_with(|| {
+			precompiles()
+				.prepare_test(
+					Alice,
+					Precompile1,
+					PCall::delegation_amount {
+						delegator: Address(Alice.into()),
+						candidate: Address(Alice.into()),
+					},
+				)
+				.expect_cost(0) // TODO: Test db read/write costs
+				.expect_no_logs()
+				.execute_returns_encoded(0u32);
+		});
+}
+
+#[test]
+fn delegation_amount_nonzero() {
+	ExtBuilder::default()
+		.with_balances(vec![(Alice.into(), 1_000), (Bob.into(), 1_000)])
+		.with_candidates(vec![(Alice.into(), 1_000)])
+		.with_delegations(vec![(Bob.into(), Alice.into(), 1_000)])
+		.build()
+		.execute_with(|| {
+			precompiles()
+				.prepare_test(
+					Alice,
+					Precompile1,
+					PCall::delegation_amount {
+						delegator: Address(Bob.into()),
+						candidate: Address(Alice.into()),
+					},
+				)
+				.expect_cost(0) // TODO: Test db read/write costs
```

Comment:
Are these TODOs to be addressed in this PR or a later one?

Reply:
later one, all staking precompile tests have this TODO so it is here because of copy/paste from other tests